code/collections.rb

```diff
@@ -1,15 +1,45 @@
 # collections
 
 # 1. Create a variable `point1` and assign it a hash representing a coordinate with keys `:x` and `:y` which have integer values
-
+point1 = {
+  x: 6,
+  y: 12
+}
 # 2. Print the y value
+puts point1[:y]
 
 # 3. Make an array, `points`, that includes `point1` and two more points (`points.length` should be 3)
+points = [
+  point1 = {
+    x: 6,
+    y: 12,
+  },
+  point2 = {
+    x: 3,
+    y: 9,
+  },
+  point3 = {
+    x: 24,
+    y: 17,
+  }
+]
```

Comment:
👍 (would work fine to not have the `point1`, `point2`, `point3` variables but because assignment evaluates to the value being assigned, this works)